src/kirby/components/checkbox/checkbox.component.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { Spectator, createTestComponentFactory } from '@netbasal/spectator';
+import { IonCheckbox } from '@ionic/angular';
+
+import { MockComponent } from 'ng-mocks';
+import * as ionic from '@ionic/angular';
+import { SimpleChange } from '@angular/core';
+
+import { CheckboxComponent } from './checkbox.component';
+
+describe('CheckboxComponent', () => {
+  const checked = true;
+  const shape = 'square';
+  const color = 'primary';
+
+  let spectator: Spectator<CheckboxComponent>;
+
+  const createHost = createTestComponentFactory({
+    component: CheckboxComponent,
+    declarations: [MockComponent(ionic.IonCheckbox)],
+  });
+
+  // beforeEach(async(() => {
+  //   TestBed.configureTestingModule({
+  //     declarations: [CheckboxComponent, MockComponent(ionic.IonCheckbox)],
+  //   }).compileComponents();
+  // }));
+
+  beforeEach(() => {
+    spectator = createHost({ checked, shape, color });
+    // fixture = TestBed.createComponent(CheckboxComponent);
```

Comment:
Remove commented out code, if you need it at a later point in time, you can retrieve it through git.